Include/abstract.h

```diff
@@ -397,13 +397,23 @@ PyAPI_FUNC(int) PyIter_Check(PyObject *);
    This function always succeeds. */
 PyAPI_FUNC(int) PyAIter_Check(PyObject *);
 
+#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x030e0000
+/* Return 1 and set 'item' to the next item of iter on success.
```

Comment:
```suggestion
/* Return 1 and set 'item' to the next item of 'iter' on success.
```

Small nitpick (Victor mentioned it somewhere else I think)